PROCESSORS.md

```diff
@@ -2449,6 +2449,7 @@ In the list below, the names of required properties appear in bold. Any other pr
 | **Multipart Upload AgeOff Interval**   | 60 min                   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | Specifies the interval at which existing multipart uploads in AWS S3 will be evaluated for ageoff. When processor is triggered it will initiate the ageoff evaluation if this interval has been exceeded.                                                                                                   |
 | **Multipart Upload Max Age Threshold** | 7 days                   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | Specifies the maximum age for existing multipart uploads in AWS S3. When the ageoff process occurs, any upload older than this threshold will be aborted.                                                                                                                                                   |
+| **Checksum Algorithm**                 | CRC64NVME                | CRC32<br/>CRC32C<br/>SHA1<br/>SHA256<br/>CRC64NVME                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | Checksum algorithm used to verify the uploaded object.                                                                                                                                                                                                                                                      |
```

Comment:
Now that this is added and the default is not MD5, can a user make minifi FIPS compliant by just enabling FIPS support and following the FIPS steps (config generation) in CONFIGURE.md?

Reply:
Yes, with the latest version with FIPS enabled no additional configuration is needed even for the S3 processors.